tools/migrate-lp-user-to-github

```diff
@@ -122,6 +119,8 @@ def add_lp_and_github_remotes(lp_user, gh_user):
             " LP repo".format(lp_user))
         lp_remote_name = 'launchpad-{}'.format(lp_user)
         subp(['git', 'remote', 'add', lp_remote_name, lp_remote])
+        log("launchpad: Pushing to ensure LP repo exists")
+        subp(['git', 'push', lp_remote_name, 'master:master'])
     subp(['git', 'fetch', lp_remote_name])
```

Comment:
Could this instead be a try except block? If we fail to git fetch, then we know we don't have one, so only then perform the git push lp_remote_name master:master

Reply:
Done and pushed.